apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -512,17 +503,6 @@
             </button>
             <% } %>
         </div>
-        <div class="column mobile center aligned tablet right aligned computer right aligned buttons tablet no-margin-right-last-child computer no-margin-right-last-child">
-            <button
-                type="submit"
-                class="ui primary large button"
-                tabindex="4"
-                role="button"
-                data-testid="login-page-continue-login-button"
-            >
-                <%=StringEscapeUtils.escapeHtml4(AuthenticationEndpointUtil.i18n(resourceBundle, "continue"))%>
-            </button>
-        </div>
```

Comment:
Why are we removing this?

Reply:
It was a bug while merging 
![image](https://user-images.githubusercontent.com/46132469/183247937-e2434c8f-ba82-41b3-a30e-451c21481356.png)